tests/integration-tests/configs/develop.yaml

```diff
@@ -66,6 +66,17 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["alinux2"]
           schedulers: ["slurm"]
+  create:
+    test_create.py::test_create_wrong_os:
+      dimensions:
+        - regions: ["eu-central-1"]
+          instances: {{ common.INSTANCES_DEFAULT_X86 }}
+          oss: ["ubuntu1804"]
```

Comment:
Can you add a comment in order to mention that this test works only with an os that is different than centos7? let's try to add this hints in the config file so that we know what we can change without having to look at every single test

Reply:
I will add it